### PR TITLE
Move brand assets to documentation

### PR DESCRIPTION
**Pull request by tammytdo**
## Fixes

Fixes https://github.com/WordPress/openverse/issues/777 by @dhruvkb 

## Description

Moves the brand directory into documentation site.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by tammytdo** [comment]
This PR references [Collect documentation inside Sphinx
[#777](https://github.com/WordPress/openverse/issues/777)](https://github.com/orgs/WordPress/projects/75/views/8?filterQuery=label%3A%22good+first+issue%22+is%3Aopen+-label%3A%22%E2%9B%94+status%3A+blocked%22+-label%3A%22%F0%9F%A7%B1+stack%3A+frontend%22+&pane=issue&itemId=21432755&sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Assignees)

**Review by sarayourfriend — CHANGES_REQUESTED** [review]
Thanks for the PR @tammytdo! Exciting to close that issue soon :)

I've gone ahead and filled out the PR description template for you, as far as I can, but please note for future reference that our workflow automations depend on the `Fixes` section being filled out.

I'm requesting changes because the Sphinx build will have errors currently, due to `brand/README.md` being an "orphan" page:

```
checking consistency... /home/sara/projects/openverse/documentation/brand/README.md: WARNING: document isn't included in any toctree
```

You can check this by running `just documentation/live`. The issue, as the warning states, is that the new README isn't included in a toctree.

This will be fixed if you rename `README.md` to `index.md`, because all `index.md` files in direct sub-directories of the documentation site's root are automatically referenced in this toctree:

https://github.com/WordPress/openverse/blob/710cda58c0fbb3636cd0a4329da181de5804d72b/documentation/index.md#L26-L31

So that would fix the warning there, and you can verify it is gone by running `just documentaiton/live` after renaming it. You shouldn't see the warning anymore, and you'll be able to visit the page locally on the live development site at the URL logged by the `just documentation/live` command.

However, I will request one more change, which is to move the brand directory into the `meta` subdirectory, so `documentation/meta/brand/`. The meta subdirectory does _not_ automatically include all subdirectory `index.md`, so you'll need to modify this toctree to explicitly reference the brand document:

https://github.com/WordPress/openverse/blob/0a9d07d8bc1797fedbac3e647c10f37a603db7f8/documentation/meta/index.md#L3-L14

I'd recommend putting it first in that list of documents.

So to summarise:
1. Rename the `README.md` to `index.md`
2. Move the brand directory under `meta` and add a reference to the `brand/index` in the toctree in `documentation/meta/index.md`

If you need any help completing these tasks or run into issues, please let us know. Thanks again, very much!

**Comment by fcoveram** [comment]
I will unassign myself as a reviewer because I can't provide proper feedback on the work made.

**Comment by tammytdo** [comment]
@sarayourfriend Updates pushed. Thank you for the images and extra clear summary!

File: archive/jars/aws-java-sdk-1.11.212.jar
Comment (AetherUnbound) [inline archive/jars/aws-java-sdk-1.11.212.jar:1]: Hmm, this file in the archive directory probably shouldn't have been removed 🤔

**Review by AetherUnbound — DISMISSED** [review]
Thank you @tammytdo! It looks like this is now failing the `CODEOWNERS` check for this reason:

```
==> Executing File Exist Checker (711.711µs)
    [err] line 32: "brand/" does not match any files in repository
```

I think this has to do with this line:

https://github.com/WordPress/openverse/blob/143ad118368b1ed003145764096ebc9c5b38e002/.github/CODEOWNERS#L32-L31

Do you mind removing it as well? Also, it looks like there was another file (a `.jar`) that was removed as part of your PR. We probably don't need it after all, but it's best to remove that in a separate PR.

**Comment by sarayourfriend** [comment]
@AetherUnbound Do you think it's worth revisiting the archived DAGs generally, at this point, and seeing if we can just remove the code altogether? If it absolutely needs to be easy to access, the old and archived `WordPress/openverse-catalog` repository would continue to have that code, but we could also zip it up and upload it somewhere if needed.

**Comment by zackkrida** [comment]
@sarayourfriend concerning the archived dags I think they should be removed, and we perhaps add a line in our documentation somewhere linking to a git commit which still contains those files.

**Comment by AetherUnbound** [comment]
Having now been several years out from the decision to go with an `archive/` directory, I think it's very safe to just remove it wholesale at this point. Part of my hesitancy in doing so was not wanting to delete major parts of the codebase without getting more context, even if they weren't being used at the time. But because it's been so long and we've shown it's really not necessary, I am fully on board with getting rid of it!

**Comment by sarayourfriend** [comment]
@tammytdo thanks again for working on this issue. Will you be able to implement the requested changes? If not, please let us know and one of the maintainers can update it with the requested changes to get it re-reviewed and merged.

**Comment by sarayourfriend** [comment]
I'm going to get this PR rebased and ready for review again. Thanks for kicking it off @tammytdo!

**Review by sarayourfriend — APPROVED** [review]

**Comment by sarayourfriend** [comment]
@dhruvkb @AetherUnbound if either of y'all can give a :+1: here, please feel free to merge after doing so.

Thanks again @tammytdo!

**Review by dhruvkb — APPROVED** [review]
LGTM! Thanks for the contribution @tammytdo!
